libultraship/libultraship/Lib/Fast3D/gfx_direct3d11.cpp

```diff
@@ -663,7 +664,13 @@ static void gfx_d3d11_draw_triangles(float buf_vbo[], size_t buf_vbo_len, size_t
         rasterizer_desc.CullMode = D3D11_CULL_NONE;
         rasterizer_desc.FrontCounterClockwise = true;
         rasterizer_desc.DepthBias = 0;
-        rasterizer_desc.SlopeScaledDepthBias = d3d.zmode_decal ? -2.0f : 0.0f;
+        if (CVar_GetS32("gN64Mode", 0) == 0) {
```

Comment:
I'm glad to see this fixed!! We probably don't want to read cvars in libultra though.. right? This cvar is specific to the oot port.

Reply:
> I'm glad to see this fixed!! We probably don't want to read cvars in libultra though.. right? This cvar is specific to the oot port.

It still has some issue onto it anyway, will find a more elgant way to do that